ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/LedgerDao.scala

```diff
@@ -161,6 +164,18 @@ trait LedgerReadDao extends ReportsHealth {
   def removeExpiredDeduplicationData(
       currentTime: Instant,
   ): Future[Unit]
+
+  /**
+    * Stops deduplicating the given command.
```

Comment:
This does not add a lot of information. I would recommend adding some context or deleting the docstring.